src/main/java/org/jabref/migrations/MergeReviewIntoComment.java

```diff
@@ -0,0 +1,70 @@
+package org.jabref.migrations;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import javafx.collections.ObservableList;
+
+import org.jabref.gui.dialogs.MergeReviewIntoCommentUIManager;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.FieldName;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MergeReviewIntoComment implements PostOpenMigration {
```

Comment:
I think most of this code should move to the logic package.

Reply:
I told him to move it there, because I want to have the migrations together. The migrations package should probably go into logic, but some of the migration classes are awefully coupled to UI classes.

Reply:
ok, then the location is fine. But at least the new class should be completely decoupled in logic and gui since otherwise the migration of the migration package is getting harder with this PR.

Reply:
@tobiasdiez I guess you mean this line?
https://github.com/JabRef/jabref/blob/b246cfe89dcd97c229dc7b2c9cc79769d6227e32/src/main/java/org/jabref/migrations/MergeReviewIntoComment.java#L38

I can create another Class for this confirmation, but where would you put that?

Reply:
I have no deep knowledge about the migration code, so the following may be a bit naïve. For me, the expected flow is something like:
- You have a general migration manager class (in gui), which is getting called as soon as the user opens a database
- This manager has a list of possible migrations which are defined in logic and which provide two methods: one for determining if an action is needed and a second one to actually perform the action
- The manager first calls `actionNeeded` and maybe asks the user for confirmation, then runs `performAction` on the migration action.

Not sure how much of this architecture is already in place and how much you are willing to refactor for this PR. My initial comment was triggered by the fact that a new class was created that mixed UI and logic.

Reply:
Okay, I totally support your point, however this is all a bit tricky, as these PostOpenMigrations are all called before the main GUI is instatiated.

Suggestion: Lets merge this PR now, and I'll refactor the code in a sensible way in a followup PR. (I'm on the train tomorrow for at least 5 hours)

Reply:
That's fine with me!

Reply:
The follow-up was done at https://github.com/JabRef/jabref/pull/3733